app/services/ship_placer.rb

```diff
@@ -16,6 +16,14 @@ def run
     end
   end
 
+  def message
```

Comment:
This is fine for now but we may want to consider making it more extendable. Lets not worry about it for now, but I'll go ahead and create a waffle card and I can always play around with it later.